tests/Resources/ti.ui.tabgroup.test.js

```diff
@@ -379,6 +374,101 @@ describe('Titanium.UI.TabGroup', function () {
 			tabGroup.addTab(tabB);
 			tabGroup.open();
 		});
+
+		describe('#close()', () => {
+			it('is a Function', () => {
+				tabGroup = Ti.UI.createTabGroup();
+
+				should(tabGroup).have.a.property('close').which.is.a.Function();
+			});
+
+			it('returns a Promise', finish => {
+				const tabA = Ti.UI.createTab({
+					title: 'Tab A',
+					window: Ti.UI.createWindow()
+				});
+				const tabB = Ti.UI.createTab({
+					title: 'Tab B',
+					window: Ti.UI.createWindow()
+				});
+				tabGroup = Ti.UI.createTabGroup();
+				tabGroup.addTab(tabA);
+				tabGroup.addTab(tabB);
+
+				const openPromise = tabGroup.open();
+				openPromise.then(() => {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.tabgroup.test.js;
  Line: 399;
-->

- :warning: tests/Resources/ti.ui.tabgroup.test.js line 399 – Each then() should return a value or throw (promise/always-return)